requests-for-comments/2022-11-02-onboarding-2.0-continued.md

```diff
@@ -0,0 +1,121 @@
+# Request for comments: Onboarding 2.0 (continued) - Emanuele Capparelli / Raquel Smith
+
+## Problem statement
+
+With this effort we're tackling a couple different problems with onboarding:
+
+1. Current activation rates for organizations are lower than we'd like. This means that too many people sign up for PostHog, but then fail to get set up properly or see value before they get tired and leave.
+2. Current activation rates for session recordings (even after successful event ingestion) are lower than we'd like. This means that people get set up, but then don't use the session recording product.
+
+### Hypotheses for some things that could cause this to happen (independently or combined)
+- Activation / Onboarding:
+  - Signup form asks for lots of information (e.g. Where did you hear about us?) which adds friction
+  - If you are a non-technical user, onboarding is a daunting experience, and inviting a team member is not exposed enough
+  - Email flows are basic and we could do with better flows to reduce churn during onboarding
+  - The onboarding flow doesn't provide value after completion, while it could be kept as reference to specific tutorials or docs in the manual
+  - The bookmarklet is currently used to skip the onboarding and skews the metrics
+  - The onboarding is pretty boring, we could do with more hedgehogs
+- Introducing users to other products:
+  - Onboarding flow does not take into account session recording or feature flags, but only event tracking
+  - We currently have an activation checklist prompt, but it has performed poorly, and we might want to integrate this directly in the onboarding flow as additional non-blocking steps we suggest users to perform
+  - We have developed a product tour but it hasn't been released to production yet, it could be added as a final optional step after the onboarding
```

Comment:
Is there a blocker here?

Reply:
It keeps on breaking down as the UI changes, I never had a chance to sit down and fix this properly as delivering the billing infrastructure was priority n. 1 recently, but that's out now so I feel like it's a good time to deliver on this